x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -33,25 +30,22 @@ export const ENGINE_ANALYTICS_RECENT_QUERIES_PATH = `${ENGINE_ANALYTICS_PATH}/re
 export const ENGINE_ANALYTICS_QUERY_DETAILS_PATH = `${ENGINE_ANALYTICS_PATH}/query_detail`;
 export const ENGINE_ANALYTICS_QUERY_DETAIL_PATH = `${ENGINE_ANALYTICS_QUERY_DETAILS_PATH}/:query`;
 
-export const ENGINE_DOCUMENTS_PATH = '/documents';
+export const ENGINE_DOCUMENTS_PATH = `${ENGINE_PATH}/documents`;
 export const ENGINE_DOCUMENT_DETAIL_PATH = `${ENGINE_DOCUMENTS_PATH}/:documentId`;
-export const getDocumentDetailRoute = (engineName: string, documentId: string) => {
-  return generatePath(ENGINE_PATH + ENGINE_DOCUMENT_DETAIL_PATH, { engineName, documentId });
-};
 
-export const ENGINE_SCHEMA_PATH = '/schema/edit';
-export const ENGINE_REINDEX_JOB_PATH = '/reindex-job/:activeReindexJobId';
+export const ENGINE_SCHEMA_PATH = `${ENGINE_PATH}/schema/edit`;
+export const ENGINE_REINDEX_JOB_PATH = `${ENGINE_PATH}/reindex-job/:activeReindexJobId`;
 
-export const ENGINE_CRAWLER_PATH = '/crawler';
+export const ENGINE_CRAWLER_PATH = `${ENGINE_PATH}/crawler`;
 // TODO: Crawler sub-pages
 
-export const META_ENGINE_SOURCE_ENGINES_PATH = '/engines';
+export const META_ENGINE_SOURCE_ENGINES_PATH = `${ENGINE_PATH}/engines`;
 
-export const ENGINE_RELEVANCE_TUNING_PATH = '/search-settings';
-export const ENGINE_SYNONYMS_PATH = '/synonyms';
-export const ENGINE_CURATIONS_PATH = '/curations';
+export const ENGINE_RELEVANCE_TUNING_PATH = `${ENGINE_PATH}/search-settings`;
+export const ENGINE_SYNONYMS_PATH = `${ENGINE_PATH}/synonyms`;
+export const ENGINE_CURATIONS_PATH = `${ENGINE_PATH}/curations`;
 // TODO: Curations sub-pages
-export const ENGINE_RESULT_SETTINGS_PATH = '/result-settings';
+export const ENGINE_RESULT_SETTINGS_PATH = `${ENGINE_PATH}/result-settings`;
 
-export const ENGINE_SEARCH_UI_PATH = '/reference_application/new';
-export const ENGINE_API_LOGS_PATH = '/api-logs';
+export const ENGINE_SEARCH_UI_PATH = `${ENGINE_PATH}/reference_application/new`;
+export const ENGINE_API_LOGS_PATH = `${ENGINE_PATH}/api-logs`;
```

Comment:
This I really like, and think we should keep regardless.  I think it makes sense that these are full paths from the app root instead of paths off a parent vieww